Project.toml

```diff
@@ -22,7 +22,6 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Bonobo = "0.1.3"
 DataStructures = "0.18"
 Distributions = "0.25"
-FrankWolfe = "0.2.11"
```

Comment:
is that normal?

Reply:
Weird, I didn't do that. It might have happened then I used my own branch for the experiments..